tools/packaging/packaging.psm1

```diff
@@ -1268,7 +1268,6 @@ function Get-PackageDependencies
                 "libc6",
                 "libgcc1",
                 "libgssapi-krb5-2",
-                "liblttng-ust0",
```

Comment:
Have you verified with the packages generated after this change can be built and installed?

Reply:
Balu verified and asked for this.

Reply:
verified this and ran on Coordinated pipeline and tested artifact deb package.